crates/biome_js_analyze/src/semantic_analyzers/style/no_var.rs

```diff
@@ -49,6 +51,14 @@ impl Rule for NoVar {
 
     fn run(ctx: &RuleContext<Self>) -> Self::Signals {
         let declaration = ctx.query();
+        let ts_global_declaratio = &declaration
+            .syntax()
+            .ancestors()
+            .find_map(TsGlobalDeclaration::cast);
+
+        if ts_global_declaratio.is_some() {
+            return None;
+        }
```

Comment:
I think we should move this check after the `is_var` check because it is more effective of checking `is_var`.